<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_menu</class>
 <widget class="QMainWindow" name="fm_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background:#ececec
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_comodato">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled{background:#00a0c0;}
QPushButton:disabled{background:#90c0c0;}
QPushButton:pressed{background:#73bebe;}
QPushButton{
color:#fff;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>Verificar
comodatos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_novaVenda">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled{background:#00a0c0;}
QPushButton:disabled{background:#90c0c0;}
QPushButton:pressed{background:#73bebe;}
QPushButton{
color:#fff;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>Nova venda
de esmaltes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_estoque">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:enabled{background:#00a0c0;}
QPushButton:disabled{background:#90c0c0;}
QPushButton:pressed{background:#73bebe;}
QPushButton{
color:#fff;
font-size:14pt;
border:none;
border-radius:10px;
}</string>
         </property>
         <property name="text">
          <string>Atualizar estoque</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_login">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:enabled{background:none;}
QPushButton:disabled{background:none;}
QPushButton:pressed{background:#c0c0c0;}
QPushButton{
font-size:10pt;
border:none;
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lb_logado">
      <property name="styleSheet">
       <string notr="true">QLabel{
background:#e0e0e0;
color:#808080;
font-size:10pt;
border:none;
border-radius:6px;
}</string>
      </property>
      <property name="text">
       <string>Logado</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>272</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGest_o">
    <property name="title">
     <string>Gestão</string>
    </property>
    <widget class="QMenu" name="menuVendedores">
     <property name="title">
      <string>Vendedores</string>
     </property>
     <addaction name="actionGestao"/>
     <addaction name="actionTransaoes"/>
    </widget>
    <widget class="QMenu" name="menuClientes">
     <property name="title">
      <string>Clientes</string>
     </property>
     <addaction name="actionGestao_2"/>
     <addaction name="actionTransaoes_2"/>
    </widget>
    <widget class="QMenu" name="menuEstoque">
     <property name="title">
      <string>Estoque</string>
     </property>
     <addaction name="actionEsmalte"/>
     <addaction name="actionBrinde"/>
     <addaction name="actionRoupa"/>
    </widget>
    <addaction name="menuVendedores"/>
    <addaction name="menuClientes"/>
    <addaction name="menuEstoque"/>
    <addaction name="actionDados"/>
   </widget>
   <widget class="QMenu" name="menuSistema">
    <property name="title">
     <string>Sistema</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuGest_o"/>
   <addaction name="menuSistema"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGestao">
   <property name="text">
    <string>Gestão</string>
   </property>
  </action>
  <action name="actionTransaoes">
   <property name="text">
    <string>Transações</string>
   </property>
  </action>
  <action name="actionGestao_2">
   <property name="text">
    <string>Gestão</string>
   </property>
  </action>
  <action name="actionTransaoes_2">
   <property name="text">
    <string>Transações</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionEsmalte">
   <property name="text">
    <string>Esmalte</string>
   </property>
  </action>
  <action name="actionBrinde">
   <property name="text">
    <string>Brinde</string>
   </property>
  </action>
  <action name="actionRoupa">
   <property name="text">
    <string>Roupa</string>
   </property>
  </action>
  <action name="actionDados">
   <property name="text">
    <string>Dados</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_novaVenda</tabstop>
  <tabstop>btn_login</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
